ci/ci_test.sh

```diff
@@ -7,6 +7,8 @@
 
 SELECTED_TEST_SUITE=$1
 
+export SPEC_OPTS="--order rand --format documentation --format CI::Reporter::RSpecFormatter"
```

Comment:
I see how documentation is good, but I wonder if makes sense to have it here? not sure if I will provide too much output for each run of jenkins job. what do you think?